### ISSUE TITLE
NTR: Propionic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8628]
New term request Propionic acid high in urine (segal_070724202731): Propanoic

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000432
- label: Elevated urine propionic acid level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: 
- def: The amount of propionic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8628
- synonyms:
